ttnn/cpp/ttnn/operations/experimental/reduction/argmax/argmax.hpp

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/run_operation.hpp"
+#include "ttnn/decorators.hpp"
+#include "ttnn/operations/core/core.hpp"
+
+namespace ttnn {
+namespace operations::experimental {
+
+namespace reduction {
+
+Tensor _argmax(const Tensor& input_t, int64_t _dim, bool all, const std::optional<MemoryConfig>& output_mem_config = std::nullopt);
```

Comment:
@KalaivaniMCW , as discussed, can these get removed and implementation moved to ExecuteArgMax::operation() in c++?

Reply:
@ayerofieiev-tt I think I will open an issue to track all of eltwise and implement the operator() changes